1.10/cli/command-reference/dcos-auth/dcos-auth-login.md

```diff
@@ -4,7 +4,7 @@ menu_order: 2
 ---
 
 # Description
-Log in to DC/OS authentication. 
```

Comment:
Yes, I did so in https://github.com/dcos/dcos-docs/pull/1177/commits/ebfacf385c232a43d552edfffdb00afb3fa2edd9.